mwapi/errors.py

```diff
@@ -36,3 +48,38 @@ class LoginError(RuntimeError):
     @classmethod
     def from_doc(cls, doc):
         return cls(doc.get('result'))
+
+
+class RequestError(requests.exceptions.RequestException):
```

Comment:
I was thinking of http://stackoverflow.com/a/6246394. You'll have to do instanceof() with inheritance but not exception wrapping.